src/preload.php.stub

```diff
@@ -42,7 +42,7 @@ foreach ($files as $file) {
             throw new \Exception($file . ' does not exist');
```

Comment:
This line will not work as expected.
<pre>
!@include_once($file) || !file_exists($file) => !@include_once($file || !file_exists($file)) // always true
</pre>